jaffle_shop/models/final/sales/fnl_sales_newcustomers.sql

```diff
@@ -0,0 +1,6 @@
+SELECT 
+  DATE_TRUNC('MONTH', customers.first_order) AS month_start_date
+  , COUNT(customers.customer_id) AS new_customer_count
+FROM {{ ref('wh_customers') }} AS customers
+GROUP BY DATE_TRUNC('MONTH', customers.first_order)
+ORDER BY DATE_TRUNC('MONTH', customers.first_order) DESC
```

Comment:
No 'order bys' :)

Reply:
Oh ok ! Yes thinking about it that makes sense as whatever uses the model can order if needed?

Reply:
We never hardcode in 'order bys' in models (downstream process can do that if necessary) this is just the because of the added processing required.
Order bys that are part of window functions are okay `ROW_NUMBER() OVER (PARTITION BY col1 ORDER BY col2)`